artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis

```diff
@@ -105,6 +105,16 @@ if [ -z "$LOG_MANAGER" ] ; then
   LOG_MANAGER="$ARTEMIS_HOME/lib/${logmanager}"
 fi
 
+# determine which native library version to load
+LIBRARY_PATH="-Djava.library.path=\"$ARTEMIS_HOME/bin/lib"
+if [ "$(uname -m)" = "x86_64" ]; then
+  LIBRARY_PATH="$LIBRARY_PATH/linux-x86_64\""
+elif [ "$(uname -m)" = "i686" ]; then
+  LIBRARY_PATH="$LIBRARY_PATH/linux-i686\""
+else
+  LIBRARY_PATH=""
```

Comment:
If the user is on a Linux system then I would expect `uname -a` to return `x86_64` or `i686`.  My thought here was to avoid setting the library path for systems where `uname -a` returned something other than what we support out-of-the-box.